cmd/commands/function.go

```diff
@@ -83,6 +84,9 @@ func FunctionCreate(fcTool *core.Client) *cobra.Command {
 				createFunctionOptions.BuildpackImage = buildpack
 			} else if invokerURL, exists := invokers[invoker]; exists {
 				createFunctionOptions.InvokerURL = invokerURL
+			} else if invoker == "custom" && createFunctionOptions.InvokerURL != "" {
```

Comment:
add a comment to indicate there is no work to do and not an oversight or duplication of the next line

Reply:
Good point. It looks suspiciously similar to line below.